samples/nfc/writable_ndef_msg/src/ndef_file_m.c

```diff
@@ -41,13 +41,13 @@ static struct nvs_fs fs = {
 	.sector_size = NVS_SECTOR_SIZE,
 	.sector_count = NVS_SECTOR_COUNT,
 	.offset = NVS_STORAGE_OFFSET,
-	.flash_device = NVS_FLASH_DEVICE,
 };
 
 int ndef_file_setup(void)
 {
 	int err;
 
+	fs.flash_device  = NVS_FLASH_DEVICE;
```

Comment:
```suggestion
	fs.flash_device = NVS_FLASH_DEVICE;
```

you also need to check if the device is not `NULL`, since `NVS_FLASH_DEVICE` uses device_get_binding underneath.